### ISSUE TITLE
Target and input mismatch?

### ISSUE THREAD

**Issue by sammyl-99** [VSainteuf/utae-paps#23]
Hi i am getting a target and input mismatch. I'm unsure how the input has gotten to this shape, as it starts with a shape (b,t,c,h,w) of [1,12,15,256,256]. Do you have any suggestions on where to look in the code?

UserWarning: Using a target size (torch.Size([1, 1, 256, 256])) that is different to the input size (torch.Size([1, 20, 256, 256])). This will likely lead to incorrect results due to broadcasting.

**Comment by VSainteuf** [comment]
Hi,
I guess you need to squeeze the second dimension of your target. 
See the [doc](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html) of the cross entropy loss implementation of PyTorch. 
Cheers
